perception/CMakeLists.txt

```diff
@@ -74,10 +77,12 @@ include_directories(SYSTEM
                     ${Boost_INCLUDE_DIRS}
                     ${GLEW_INCLUDE_DIR}
                     ${GLUT_INCLUDE_DIR}
+                    ${X11_INCLUDE_DIR}
                     )
 
 link_directories(${Boost_LIBRARY_DIRS})
 link_directories(${catkin_LIBRARY_DIRS})
+link_directories(${X11_LIBRARY_DIR})
```

Comment:
Better. Could you please try to replace ${X11_INCLUDE_DIR} by ${OPENGL_INCLUDE_DIR}?